test/proxy.spec.js

```diff
@@ -0,0 +1,23 @@
+import { bootstrap } from '../lib/proxy';
+
+describe('proxy', () => {
```

Comment:
These tests are now somehow useless because we don't do much. I still wanted to make sure that the bootstrapping code is called correctly.